src/utils/filterContests.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+import type { Contest } from '../types';
+
+export function filterContests(contests: Array<Contest>, from?: Date, to?: Date): Array<Contest> {
```

Comment:
The behaviour of this function should be documented, it's not at all obvious. Also, I think the name is not very good, it should clarify what kind of filtering is done. Maybe `filterStartTime` is better. It is clear from the type/name of the arguments that it filters an array of contests, but it is not clear that is only considers the startTime to do so.